kernel/src/client/arrow_data.rs

```diff
@@ -321,22 +325,23 @@ mod tests {
     use crate::actions::Metadata;
     use crate::DeltaResult;
     use crate::{
-        actions::schemas::log_schema,
-        simple_client::{data::SimpleData, SimpleClient},
-        EngineData, EngineInterface,
+        actions::schemas::log_schema, client::sync::SyncEngineInterface, EngineData,
+        EngineInterface,
     };
```

Comment:
nit: probably cleaner to just have three import lines?

In general, I have no intuition for when it's good to get fancy with the `{}` nesting and multiline , vs better to prefer more-but-simpler imports... but here there's a lot of redundancy that could be eliminated by e.g.
```rust
use crate::actions::{schemas::log_schema, Metadata};
use crate::simple_client::{data::SimpleData, SimpleClient};
use crate::{DeltaResult, EngineData, EngineInterface};
```
or I suppose:
```rust
use crate::{
  actions::{schemas::log_schema, Metadata},
  simple_client::{data::SimpleData, SimpleClient},
  DeltaResult, EngineData, EngineInterface,
};
```

Reply:
Yeah. I have a bad habit of just letting eglot import things for me, and it doesn't seem to always do the most clean way of doing things. I wish `cargo fmt` was more opinionated about this, but regardless, I've cleaned this up. thanks